### ISSUE TITLE
Changelog Text Unreadable

### ISSUE THREAD

**Issue by SensehacK** [SensehacK/sensehack.github.io#77]
**Describe the bug**
Good color contrast should be present.

**To Reproduce**

**Expected behavior**

**Screenshots**
![Screen Shot 2020-04-14 at 12 42 26 AM](https://user-images.githubusercontent.com/8469205/79186884-d35a0c00-7de8-11ea-9cfe-75033fc9f61a.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
